### PR TITLE
Add an alert before deleting resources via the bulk delete #11405

### LINKED ISSUE #11405: Bulk Data Manager - Data Deletion function cannot be rolled back, needs more explicit language to alert users
Once a bulk deletion task is submitted it cannot be rolled back like the other tasks in the Bulk Data Manager. This is not apparent to the user until after the task is started and completed.

It seems as though the user should be alerted to the inability to rollback the task before hitting the start button.
Is there a manual way to roll back the deletion, perhaps via Celery task management? Or is this a full deletion method? The objects deleted were in this example were tiles. 

image

### PR DESCRIPTION
**Pull request by MikeGriniezakis**
### Types of changes

-   [x] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Description of Change

Added an alert message before running the bulk deletion. This was done to prevent unwanted deletion of data by
requiring confirmation before executing the task. 

### Issues Solved

Closes #11405

### Checklist

-   I targeted one of these branches:
    - [ ] dev/8.0.x (under development): features, bugfixes not covered below
    - [x] dev/7.6.x (main support): regressions, crashing bugs, security issues, major bugs in new features
    - [ ] dev/6.2.x (extended support): major security issues, data loss issues
-   [x] I added a changelog in arches/releases
-   [ ] I submitted a PR to arches-docs (if appropriate)
-   [ ] Unit tests pass locally with my changes
-   [ ] I added tests that prove my fix is effective or that my feature works
-   [ ] My test fails on the target branchT

#### Ticket Background
*   Tested by: @AdvanceServices

### CODE REVIEW

File: arches/app/media/js/views/components/etl_modules/bulk-data-deletion.js [hunk 1]
```diff
@@ -144,21 +144,25 @@ define([
             }
         });
 
+        this.deleteAlert = function() {
+            self.alert(
+                new AlertViewModel(
+                    'ep-alert-red',
+                    "Delete Data",
```
Comment (apeters) [inline arches/app/media/js/views/components/etl_modules/bulk-data-deletion.js:151]: Let's change this to blue as it's informational
```suggestion
                    'ep-alert-blue',
```

**Review by apeters — CHANGES_REQUESTED** [review]

File: arches/app/media/js/views/components/etl_modules/bulk-data-deletion.js [hunk 2]
```diff
@@ -144,21 +144,25 @@ define([
             }
         });
 
+        this.deleteAlert = function() {
+            self.alert(
+                new AlertViewModel(
+                    'ep-alert-red',
```
Comment (apeters) [inline arches/app/media/js/views/components/etl_modules/bulk-data-deletion.js]: Let's change this to "ep-alert-blue" as it's informational.

**Review by apeters — CHANGES_REQUESTED** [review]

File: arches/app/media/js/views/components/etl_modules/bulk-data-deletion.js (hunk 2 above)
Comment (MikeGriniezakis) [reply arches/app/media/js/views/components/etl_modules/bulk-data-deletion.js]: Changed in latest commit d7537f4327a41714fdb86bb64a9b79ee43543fea

File: arches/app/media/js/views/components/etl_modules/bulk-data-deletion.js [hunk 3]
```diff
@@ -144,21 +144,25 @@ define([
             }
         });
 
+        this.deleteAlert = function() {
+            self.alert(
+                new AlertViewModel(
+                    "ep-alert-blue",
+                    "Delete Data",
+                    "This action will delete the selected records permanently. Are you sure you want to proceed?",
```
Comment (apeters) [inline arches/app/media/js/views/components/etl_modules/bulk-data-deletion.js]: these strings should be made localizable by adding entries into `javascript.htm` and then reference those using the `arches.translations....` object.

**Review by apeters — CHANGES_REQUESTED** [review]

File: arches/app/media/js/views/components/etl_modules/bulk-data-deletion.js (hunk 3 above)
Comment (MikeGriniezakis) [reply arches/app/media/js/views/components/etl_modules/bulk-data-deletion.js]: The strings are localized in this commit affeef8b2763cfb2144d2cf3800917209223fa57

**Review by apeters — APPROVED** [review]
Looks good!

**Pull request merged**